onewire/__init__.py

```diff
@@ -388,23 +390,28 @@ def _sensor_cycle(self):
                     value = self.read('/uncached' + path).decode()
                     if key.startswith('T') and value == '85.0000':
                         self.logger.info("1-Wire: problem reading {0}. Wiring problem?".format(addr))
+                        temperror=True
```

Comment:
``temperror`` braucht es IMHO hier gar nicht: wenn der Temperaturfehler auftritt, dann wird mit dem nachfolgenden ``continue`` in der for-loop das nächste Element gewählt. Daher braucht auch weiter unten nicht auf temperror getestet werden.